Veneer/Classes/UIViewController+Veneer.swift

```diff
@@ -27,7 +27,7 @@ extension DismissType: CustomStringConvertible {
 
 public extension UIViewController {
     
-    public func showVeneer(withHighlight highlight: Highlight, overlayViewType: VeneerOverlayView.Type = VeneerOverlayView.self) {
+    public func showVeneer(withHighlight highlight: Highlight, overlayViewType: VeneerOverlayView.Type = VeneerOverlayView.self, configurationCallback: ((VeneerOverlayView) -> ())? = nil) {
```

Comment:
should the callback receive an instance of `overlayViewType`?

Reply:
Yes, but you can't use a method argument in a different method parameter.

Reply:
true would need to be generic i guess, but then would lose the default value e.g. 

```
public func showVeneer<T: VeneerOverlayView>(withHighlight highlight: Highlight, overlayViewType: T.Type = T.self, configurationCallback: ((T) -> ())? = nil)
```

We specify the overlay view type in every case at the moment anyway